qubesusbproxy/core3ext.py

```diff
@@ -642,9 +642,10 @@ async def on_device_attach_usb(self, vm, event, device, options):
                 else:
                     # TODO: sanitize and include stdout
                     sanitized_stderr = ''.join(
-                        [chr(c) for c in e.stderr if 0x20 <= c < 0x80])
+                        [chr(c) for c in e.stderr if 0x20 <= c < 0x80
+                         or c in [0x0A, 0x0D]])
```

Comment:
I don't think simply allowing newlines in an exception message is a good idea. Depending on what applications will do with that, it may mess with formatting, or even allow injecting something. This is especially important since it's about a message received from a VM - that's why it's sanitized after all.
Maybe better to replace disallowed chars with something (we use "_" for that in several places) instead of just removing them? Or replace newline specifically with a coma?

Reply:
> Or replace newline specifically with a coma?

Very well said. But [coma+space](https://www.sussex.ac.uk/informatics/punctuation/comma) (or semicolon+space.). Amendment will follow shortly.